lib/statesman/guard.rb

```diff
@@ -3,10 +3,17 @@
 
 module Statesman
   class Guard < Callback
+    attr_reader :name
+
+    def initialize(options = { from: nil, to: nil, callback: nil })
```

Comment:
Is there a nicer way to do this which doesn't make the `#initialize` of `Guard` as tied to that of `Callback?`